### ISSUE TITLE
Create issue templates

### ISSUE THREAD

**Issue by kevtan** [stanford-ssi/sequoia-software#12]
It would be helpful to create issue templates to improve and standardize the contributing experience. I'd like to open up the discussion here to see what kinds of issues everyone would like to see. GitHub comes with two default templates, **Bug report** and **Feature request** that seem relevant, but @polygnomial @lgnashold are there any SSI specific issue templates you'd like to see?

**Comment by polygnomial** [comment]
Yes, I think we probably want something geared towards **testing**. What I mean is, we eventually want some integrated testing (i.e. not unit tests) that may cut across **features**. 

Otherwise, I think we should have instructions for how we plan to use **Feature** requests—specifically, requirement the feature maps to on our software requirements table.

**Comment by polygnomial** [comment]
@lgnashold @austrian-code-wizard any ideas?

**Comment by polygnomial** [comment]
I've also thought about it and I think I want a `feature` tag for sorting purposes that will track the top level requirements from our systems reqs spreadsheet.
